pages/common/aws-s3api.md

```diff
@@ -0,0 +1,36 @@
+# aws s3api
+
+> Create and delete Amazon S3 buckets and edit bucket properties.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3api/index.html>.
+
+- Create a bucket:
+
+`aws s3api create-bucket --bucket {{bucket_name}}`
+
+- Delete a bucket:
+
+`aws s3api delete-bucket --bucket {{bucket_name}}`
+
+- List buckets:
+
+`aws s3api list-buckets`
+
+- List the objects inside of a bucket and only show each object's key and size:
+
+`aws s3api list-objects --bucket {{bucket_name}} --query '{{Contents[].{Key: Key, Size: Size}}}'`
+
+- Add an object to a bucket:
+
+`aws s3api put-object --bucket {{bucket_name}} --key {{object_key}} --body {{path/to/file}}`
+
+- Download object from a bucket (The output file is always the last argument):
+
+`aws s3api get-object --bucket {{bucket_name}} --key {{object_key}} {{path/to/output_file}}`
+
+- Apply an Amazon S3 bucket policy to a specified bucket:
+
+`aws s3api put-bucket-policy --bucket {{bucket_name}} --policy file://{{path/to/bucket_policy.json}}`
+
+- Download the Amazon S3 bucket policy from a specified bucket:
+
+`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output {{json|text|table|yaml|yaml-stream}} > {{path/to/bucket_policy.json}}`
```

Comment:
If I'd use for example `--output yaml`, I wouldn't get a `.json` file, right? If so, we should probably just say
```suggestion
`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output {{json|text|table|yaml|yaml-stream}} > {{path/to/bucket_policy}}`
```